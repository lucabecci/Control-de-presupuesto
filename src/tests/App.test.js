import React from 'react'
import App from '../App'
import { shallow } from 'enzyme'
import '@testing-library/jest-dom' //importacion para tener snippets de jest


describe('Test en <App />', () => {
    let wrapper = shallow( <App/> )
    test('snapshot con enzyme', () => {
        expect(wrapper).toMatchSnapshot();
    })
    
})
