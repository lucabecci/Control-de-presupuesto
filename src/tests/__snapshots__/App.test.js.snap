// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test en <App /> snapshot con enzyme 1`] = `ShallowWrapper {}`;
